### PR TITLE
Update documentation for chapter title

### PR DESCRIPTION
**Pull request by cehsu**
These changes update the schemaform documentation to describe that a function that can be used to specify a title, rather than a string, when a distinct review page title is needed.

### CODE REVIEW

**Review by jbalboni — APPROVED** [review]

File: docs/schemaform/form-config.md
Comment (markgreenburg) [inline docs/schemaform/form-config.md]: Is this true? I thought `onReview` was only for chapter titles. Maybe you meant to put this there instead of on the form title documentation?

File: docs/schemaform/form-config.md
Comment (markgreenburg) [inline docs/schemaform/form-config.md]: Up to you, but I kinda like the format for the page title - i.e., listing both types of title like:
```
 // The title of the chapter
 title: '',
 // This can also be a function that receives an onReview=true parameter only when on the review page
 title: formData => `A title for ${formData.thing}`,
```

Your call 🤷‍♂️

Comment (markgreenburg) [reply docs/schemaform/form-config.md]: On further thought, this is probably an unnecessary / opinionated nit on my part.

**Review by markgreenburg — COMMENTED** [review]
I think you might've meant to update the documentation for chapter `title` instead of form `title` - happy to approve if I'm wrong or if you can update.

**Review by markgreenburg — APPROVED** [review]

**Pull request merged**
